File: C/String/program24.c
//Accept two string from user and compare that strings

#include<stdio.h>
#include<stdbool.h>

bool strcmpX(char *src, char *dest)
{
    while((*src != '\0') && (*dest != '\0'))
    {
        if(*src != *dest)
        {
            break;
        }
        src++;
        dest++;
    }
    if((*src == '\0') && (*dest == '\0'))
    {
        return true;
    }
    else
    {
        return false;
    }

}

int main()
{
    char Arr[20];
    char Brr[20];
    bool bRet;

    printf("Enter first string: \n");
    scanf("%[^'\n']s",Arr);

    printf("Enter second string: \n");
    scanf(" %[^'\n']s",Brr);

    bRet = strcmpX(Arr,Brr);
    if(bRet == true)
    {
        printf("Strings are equal \n");
    }
    else
    {
        printf("Strings are not equal\n");
    }

    return 0;
}

File: Java/String/program24.java
//Accept two string from user and concat first string after the second string

import java.util.*;

class Stringx
{
    public String str1;
    public String str2;

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter first string");
        str1 = sobj.nextLine();

        System.out.println("Enter second string");
        str2 = sobj.nextLine();
    }

    public void strcatX()
    {
        String str3 = str1.concat(str2);

        System.out.println("String after concatination : "+str3);
        
    }
}

class program24
{
    public static void main(String arg[])
    {
        Stringx obj = new Stringx();

        obj.Accept();

        obj.strcatX();
    }
}